test/mml_builder.js

```diff
@@ -656,7 +656,8 @@ suite('mml_builder use_workers=' + useWorkers, function() {
   });
 
   // See https://github.com/Vizzuality/grainstore/issues/62
-  test('throws useful error message on invalid text-name', function(done) {
+  // skiping since carto#0.16.3 does not throw error
+  test.skip('throws useful error message on invalid text-name', function(done) {
```

Comment:
In docs: https://github.com/mapbox/carto/blob/0f8613fb60f8bfc834d560dcfb69a603f933e836/docs/2.1.1.md#text-name-expression

and

https://github.com/mapbox/carto/blob/0f8613fb60f8bfc834d560dcfb69a603f933e836/docs/2.1.1.md#string

there are some advanced uses of `text-name`, i.e: https://github.com/mapbox/carto/blob/516289b8d13f4a909ff48c2916a7f07143e1c90a/test/rendering/field_advanced.mss#L2

https://github.com/mapbox/carto/blob/516289b8d13f4a909ff48c2916a7f07143e1c90a/test/rendering/var_concat.mss#L5

I think this is why _flat strings are ok_